api/sip/client.py

```diff
@@ -825,7 +826,7 @@ class MockSIPClient(SIPClient):
     """
 
     def __init__(self, login_user_id=None, login_password=None, separator="|",
-                 target_server=None, target_port=None, location_code=None):
+                 target_server=None, target_port=None, location_code=None, institution_id=''):
```

Comment:
Here you take `institution_id` as an argument but you don't do anything with it. It should be passed into the superclass constructor along with login_user_id and separator.

Reply:
Ah, I saw I needed the value in the constructor, but missed passing it to the parent class. About that, an educational point: are we essentially overriding __init__ to mask the need for target_server and target_port in the calls to MockSIPClient? And, if so, are the parameters `target_server=None, target_port=None,` needed in the constructor?

Reply:
Right on both counts. We don't want MockSIPClient to actually make any calls, and it's probably not necessary for the constructor to take the arguments.